techempower/scripts/perf/tfb-run.sh

```diff
@@ -362,11 +375,16 @@ function run_wrk_workload() {
 	IP_ADDR=$1
 	RESULTS_LOG=$2
 	# Run the wrk load
-	echo "Running wrk load with the following parameters" >> ${LOGFILE}
-#	cmd="${HYPERFOIL_DIR}/wrk2.sh --latency --threads=${THREAD} --connections=${CONNECTIONS} --duration=${DURATION}s --rate=${REQUEST_RATE} http://${IP_ADDR}/db"
-	cmd="${HYPERFOIL_DIR}/wrk.sh --latency --threads=${THREAD} --connections=${CONNECTIONS} --duration=${DURATION}s http://${IP_ADDR}/db"
-	echo "CMD = ${cmd}" >> ${LOGFILE}
-	${cmd} > ${RESULTS_LOG}
+	if [[ ${MODE} == "monitoring" ]]; then
```

Comment:
Please update the read me with monitoring mode details.

Reply:
@kusumachalasani - TFB database was not started by the script, please check.

Also the hyperfoil zip file name has to be updated for version 0.16, it is now - https://github.com/Hyperfoil/Hyperfoil/releases/download/release-0.16/hyperfoil-0.16-SNAPSHOT.zip